### PR TITLE
Feature/#48/firebase setup

### PR DESCRIPTION
**Pull request by Exiledz5155**
## About this PR

#48 

## How to Review

cd web-app, npm i and npm run dev

From the default home URL /, click on the gibberle text. This should redirect you to /dev/sandbox

Here, ensure that nothing is displayed except the Sandbox text and a sign in button.

Ensure sign-in with google works, and once signed in, this should be displayed, (the exact documents displayed may vary):

image

Next:

- ensure add input works, adding a text input
- ensure submitting the document works, make sure to click the refresh button to see changes
- ensure the random button works, this should fill in a varying amount of inputs with random strings
- ensure that upon sign out, nothing is visible again

### Make sure to download .env in documentations channel on discord in root of the project

### CODE REVIEW

**Comment by CalvinDudd** [comment]
Having issues after clicking on "Gibberle" on the / page.

Looking into the fix now.

**Comment by CalvinDudd** [comment]
@Exiledz5155 We are still blocked on this PR, we will need to meet about this issue.

**Review by marcus-gustafson — APPROVED** [review]
lgtm, I am assuming one of you fixed the auth issue since it works now

File: web-app/src/components/dev/FirestoreDisplay.tsx
Comment (CalvinDudd) [inline web-app/src/components/dev/FirestoreDisplay.tsx:38]: nit pick: remove space in tag

**Comment by CalvinDudd** [comment]
env in roiot
Screenshot_1

**Review by CalvinDudd — APPROVED** [review]
Left a comment with the details of my review.

**Pull request merged**
